### PR TITLE
remove long in torchloss

### PR DESCRIPTION
**Pull request by qiuxin2012**
Because TorchLoss's input from scala are Float tensor currently, so for the loss like CrossEntropyLoss who require long-type target, we can use a lossFunction to wrap the Loss:
```
def lossFunc(input, target):
    return nn.CrossEntropyLoss().forward(input, target.flatten().long())
```

### CODE REVIEW

**Comment by qiuxin2012** [comment]
jenkins passed
